src/cloudsc_gpu/dwarf_cloudsc_gpu.F90

```diff
@@ -33,17 +33,30 @@ PROGRAM DWARF_CLOUDSC
 USE CLOUDSC_DRIVER_GPU_OMP_SCC_HOIST_MOD, ONLY: CLOUDSC_DRIVER_GPU_SCC_HOIST
 #endif
 
+#ifdef CLOUDSC_GPU_SCC_FIELD
```

Comment:
Split this into a `HAVE_FIELD_API` bit for the field-api imports and a `CLOUDSC_GPU_SCC_FIELD` for the specific variant?